README.md

```diff
@@ -1,26 +1,56 @@
 
-# About
+# lsif-util
 
-Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers. Featuring:
+Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers.
 
-* Validation
-* Graph visualization
-* Quick search
+## What's new
 
-# Getting Started
+* _Up to date!_ Support for [JSON Lines](http://jsonlines.org/) with the `--inputFormat` tag
+* _Cleaning up!_ Standard filtering for all tools (no more unnecessary searching)
 
-* `> git clone this repository`
-* `> cd lsif-util`
-* `> npm install`
-* `> npm run compile`
+## What's next
 
-# Validation
+* _We are moving!_ The lsif-util tools will soon migrate to [lsif-node](https://github.com/microsoft/lsif-node)
+* _Getting official!_ Global npm package is on the way
 
-`> node .\lib\validate.js [options]`
+## Usage
```

Comment:
Since usage in CI is a popular use-case, can we have a separate section for that? It seems to me that the filtering parameters will not be useful in the CI scenario, and that "usage section" can be shorter and easier to grok quickly

Reply:
I like your idea of separating filtering to another section, specially since it is not necessary (at least not for validation). Which brings me to the question: should we visualize the whole graph if no filters are entered?

Reply:
I don’t think the whole graph will be useful since Graphviz won’t be able to visualize it properly. We can throw an error of sorts to call filters out as a requirement for now - can change once we have better visualization

Reply:
Never mind me, we already visualize the whole thing if no filters are entered! Crazy, huh.

Reply:
I tried running in a very small test repo just to see what would happen. The output was the following monstrosity:

![image](https://user-images.githubusercontent.com/45983213/58600442-7dfd2680-8239-11e9-94ba-de086703092e.png)

When I tried a larger input (this repo itself, for example) it just broke. Maybe I should add a note about this in the README.